test_programs/plonky2_prove_failure/signed_lt_arr/Prover.toml

```diff
@@ -1,15 +1,15 @@
 # all combinations of the low 4 bits, i.e. 0000xxxx in binary
-x1 = [0, -1, -2, -3, -4, -5, -6, -7, -8, -9,-10,-11,-12,-13,-14,-15]
-y1 = [0, -1, -2, -3, -4, -5, -6, -7, -8, -9,-10,-11,-12,-13,-14,-15]
+x1 = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14,15]
+y1 = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9,10,11,12,13,14,15]
 # all combinations of the high 4 bits, i.e. xxxx0000 in binary
-x2 = [0,-16,-32,-48,-64,-80,-96,-112,-128,-144,-160,-176,-192,-208,-224,-240]
-y2 = [0,-16,-32,-48,-64,-80,-96,-112,-128,-144,-160,-176,-192,-208,-224,-240]
+x2 = [-16,-32,-48,-64,-80,-96,-112,-128,112,96,80,64,48,32,16,0]
```

Comment:
The are the correct numbers, but in the wrong order. These must be in sorted order with regards to signed comparison. The correct order is:
-128,-127,-126,-125,-64,-63,-62,-61,0,1,2,3,64,65,66,67